ete3 cannot be used with Python 3.13 due to the deprecation of the cgi module

The `cgi` module, used in `ete3/webplugin/webapp.py` was deprecated in Python 3.11 and removed in 3.13 (see https://docs.python.org/3.13/whatsnew/3.13.html). So trying `from ete3 import Tree` results in an import exception:

```
Traceback:
../../.local/share/uv/python/cpython-3.13.1-macos-aarch64-none/lib/python3.13/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test/test_trees.py:2: in <module>
    from ete3 import Tree
.venv/lib/python3.13/site-packages/ete3/__init__.py:56: in <module>
    from .webplugin.webapp import *
.venv/lib/python3.13/site-packages/ete3/webplugin/webapp.py:44: in <module>
    import cgi
E   ModuleNotFoundError: No module named 'cgi'
```

This can be fixed by using `urllib.parse.parse_qs`. Pull request coming up.

--- a/ete3/webplugin/webapp.py
+++ b/ete3/webplugin/webapp.py
@@ -41,7 +41,7 @@ from __future__ import print_function
 import sys
 import os
 import time
-import cgi
+from urllib.parse import parse_qs
 from hashlib import md5
 import six.moves.cPickle
 from six.moves import map
@@ -259,9 +259,9 @@ class WebTreeApplication(object):
         path = environ['PATH_INFO'].split("/")
         start_response('202 OK', [('content-type', 'text/plain')])
         if environ['REQUEST_METHOD'].upper() == 'GET' and  environ['QUERY_STRING']:
-            self.queries = cgi.parse_qs(environ['QUERY_STRING'])
+            self.queries = parse_qs(environ['QUERY_STRING'])
         elif environ['REQUEST_METHOD'].upper() == 'POST' and environ['wsgi.input']:
-            self.queries = cgi.parse_qs(environ['wsgi.input'].read())
+            self.queries = parse_qs(environ['wsgi.input'].read())
         else:
             self.queries = {}
 

